gnome-initial-setup/pages/timezone/gis-timezone-page.c

```diff
@@ -474,8 +474,13 @@ gis_timezone_page_init (GisTimezonePage *page)
 void
 gis_prepare_timezone_page (GisDriver *driver)
 {
-  gis_driver_add_page (driver,
-                       g_object_new (GIS_TYPE_TIMEZONE_PAGE,
-                                     "driver", driver,
-                                     NULL));
+  if (gis_driver_is_live_session (driver))
+    return;
+
+  GisPage *page;
```

Comment:
Nitpick, but if you leave the `g_object_new()` call the way it was, this patch becomes a two-liner.